src/confluent/azext_confluent/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.isExperimental": false,
```

Comment:
```suggestion
```
`azext.isExperimental` is deprecated, please refer to https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#backward-compatibility-of-extension-version-tags

Reply:
I have changed this

Reply:
yes, but
> For all stable extension releases, neither azext.isPreview nor azext.isExperimental can be added into module metadata.